docs/en/04_Changelogs/4.8.0.md

```diff
@@ -10,47 +10,58 @@
 
 ### Support for silverstripe/graphql v4 {#graphql-v4}
```

Comment:
I wanted a smaller introductory section that just gives you the top line information you need to know. I took most of the detail info and move it to the relevant sub-headers